LaunchDarklyAndroidBinding/packages.config

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="GoogleGson" version="2.8.1" targetFramework="monoandroid80" />
```

Comment:
Same as iOS, my only question is whether this project should be .NET standard instead of PCL. I think it is ok though.